app/models/facility.rb

```diff
@@ -242,4 +242,8 @@ def teleconsultation_phone_numbers_with_isd
   def discardable?
     registered_patients.none? && blood_pressures.none? && blood_sugars.none? && appointments.none?
   end
+
+  def syncable_patients
+    registered_patients.with_discarded
```

Comment:
We can do the other magic things in `Facility`, `FacilityGroup` and `Organization` as well for consistency, but I've left them as they were before, since I figured that Region is effectively acting as the special-case sync class (for blocks).

Don't have a good answer to this.

Reply:
Can you help me understand if we still need to do some updating here? The code here is inconsistent with our intended "patients in a block" definition, right?

Reply:
Update: I see the `region` does the heavy lifting now. We can... delete these methods from facility and facility group?

Reply:
Hmm, in that case, it seems to me like we should remove them for a couple of reasons:
* It seems overly defensive to have these methods in here
* Regions are carrying their own weight quite well, minimizing the chances that a facility or facility group will need to be used

What do you think?

Reply:
Agree w/ @harimohanraj89, removing this kind of duplicated code and letting Regions handle it was one of the big reasons for them. 

Lets ax any code that isn't being called. (blocking change)

Reply:
Agree. Will ax these👍